src/ontology/mondo.Makefile

```diff
@@ -819,4 +819,59 @@ update-%-mappings: $(TMPDIR)/new-exact-matches-%.owl
 		convert -f obo --check false -o tmp/$(SRC)
 		mv tmp/$(SRC) $(SRC)
 		make NORM
-		mv NORM $(SRC)
\ No newline at end of file
+		mv NORM $(SRC)
+
+######################################
+##### New regeneration process #######
+######################################
+# Remove logical axioms
+tmp/mondo-no-logical-axioms.owl: mondo.owl
+	robot filter --input $< --axioms logical --output $@
+
+# Declare version and resources of the `mondo-ingest` release
+VERSION = v2023-06-14
+EXTERNAL_ONTOLOGIES:= doid ncit omim
+get-external-resources: $(addprefix tmp/,$(addsuffix .owl,$(EXTERNAL_ONTOLOGIES)))
+
+tmp/%.owl:
+	wget "https://github.com/monarch-initiative/mondo-ingest/releases/download/$(VERSION)/$*.owl" -O $@
+
+ONTOLOGIES:= $(EXTERNAL_ONTOLOGIES)
+ONTOLOGIES += mondo-no-logical-axioms
+
+# Merge the OWL files.
+tmp/merged.owl: $(addprefix tmp/,$(addsuffix .owl,$(ONTOLOGIES)))
+	robot merge $(foreach file,$^,--input $(file)) --output $@
+
+# filter mondo.sssom.tsv
+
+PREFIXES := DOID NCIT OMIM
+
+tmp/mondo-doid-omim-ncit.sssom.tsv:
+	sssom filter $(addprefix --object_id ,$(addsuffix :%,$(PREFIXES))) $(MAPPINGSDIR)/mondo.sssom.tsv -o $@
+	sssom parse $@ -o $@
+
+tmp/combined.ptable.tsv: tmp/mondo-doid-omim-ncit.sssom.tsv
+	sssom ptable --default-confidence 0.95 $< -o $@
+
+boomer-prepare-dirs:
+	mkdir -p tmp/boomer_output
```

Comment:
I tend to put this nowadays in the first row of the relevant goal, i.e. `boomer`.